### PR TITLE
HDDS-9958. Improve log for container not found in CloseContainerCommandHandler

### PR DESCRIPTION
**Pull request by DaveTeng0**
## What changes were proposed in this pull request?
Currently CloseContainerCommandHandler reports error in log when its ContainerController.getContainer() returns null. This was probably a container that never got created as documented in [HDDS-9550](https://issues.apache.org/jira/browse/HDDS-9550)/[HDDS-9600](https://issues.apache.org/jira/browse/HDDS-9600). Thus we'd like to change [this log message](https://github.com/apache/ozone/blob/52236b152ba4529cf3050d2baee4d50c6722d73b/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/CloseContainerCommandHandler.java#L105) to info with a note that the container may not have been created. (currently the error log stands out when it is probably not a real issue.) We could revisit this log later when issue [HDDS-9600](https://issues.apache.org/jira/browse/HDDS-9600) is resolved.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-9958

## How was this patch tested?
original CI pipeline tests.

### CODE REVIEW

**Comment by DaveTeng0** [comment]
cc. @sumitagrawl  @errose28

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/CloseContainerCommandHandler.java [hunk 1]
```diff
@@ -102,7 +102,12 @@ public void handle(SCMCommand command, OzoneContainer ozoneContainer,
...
 
         if (container == null) {
-          LOG.error("Container #{} does not exist in datanode. "
+          // Change the log level from error to info,
+          // since the container may not have been created
+          // due to issue documented in HDDS-9550/HDDS-9600.
+          // Currently the error log stands out when it is
+          // probably not a real issue.
+          LOG.info("Container #{} does not exist in datanode. "
               + "Container close failed.", containerId);
```
Comment (errose28) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/CloseContainerCommandHandler.java:111]: Let's try to keep the log message self documenting so it's easier for people looking at logs without the code.
```suggestion
          LOG.info("Container #{} does not exist in datanode. Its pipeline may have closed before it was created."
              + "Container close failed.", containerId);
```

**Comment by SaketaChalamchala** [comment]
@swamirishi can you please take a look?

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/CloseContainerCommandHandler.java (hunk 1 above)
Comment (DaveTeng0) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/CloseContainerCommandHandler.java]: sounds good! thanks @errose28 , updated.

**Review by errose28 — APPROVED** [review]
Thanks for the update @DaveTeng0

**Comment by adoroszlai** [comment]
Thanks @DaveTeng0 for the patch, @errose28 for the review.

**Pull request merged**